source/test/TestApp/PlayFabEventTest.h

```diff
@@ -47,6 +47,13 @@ namespace PlayFabUnit
             void LambdaCallbackTest(TestContext& testContext);
             void PrivateMemberCallbackTest(TestContext& testContext);
 
+            void BasicMultiThreadedTest(TestContext& testContext);
+
+            // We need to make sure these are able to pass within our TIMEOUT limits.
```

Comment:
>TIMEOUT [](start = 70, length = 7)

nit: is there a reason this is all caps? does it correlate to some constant somewhere? seems a bit weird to me but maybe it's communicating something I'm not seeing